src/app/routes/api/v3/sms/sms.routes.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express'
```

Comment:
I don't think we should add a new `/sms` endpoint just for this. Would the endpoint not be better suited in `/api/v3/forms/:formId/verified-sms/count/free`?

Reply:
sure, i just thought that it made more sense to be in sms cos the `form` or `/admin/form` endpoints were concerned with form properties but i'll change it! 

will change to `/admin/form` because it's regarding privileged information and also because ti should be protected by authentication (checked wtih sonjia on this)